src/Features/Core/Portable/Completion/Providers/AbstractMemberInsertingCompletionProvider.cs

```diff
@@ -56,38 +56,11 @@ public override async Task<CompletionChange> GetChangeAsync(Document document, C
             }
 
             var changes = await newDocument.GetTextChangesAsync(document, cancellationToken).ConfigureAwait(false);
-            var change = Collapse(newText, changes.ToList());
+            var change = Utilities.Collapse(newText, changes.ToImmutableArray());
```

Comment:
is CompletionUtilites not a good one to use rather than creating new Utilities helper type?

Reply:
in this case, the helper isn't really completion specific.  it's jsut a way to make a single change out of a set of changes.